sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -583,6 +583,17 @@ case class AlterTable(
   }
 }
 
+/**
+ * Show tables.
+ */
+case class ShowTables(
+    catalog: TableCatalog,
+    ident: Identifier,
+    pattern: Option[String]) extends Command {
+  override val output = ShowTablesSchema.attributes()
+
+  override lazy val schema = ShowTablesSchema.schema
```

Comment:
This defaults to `StructType.fromAttributes(output)` in QueryPlan, so no need to add it here.

Reply:
Removed.